deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -341,6 +341,14 @@ void QmlStreamer::Impl::_onWheeled( double x_, double y_, double deltaY )
     QCoreApplication::postEvent( _quickWindow, e );
 }
 
+void QmlStreamer::Impl::_onPinched( double x_, double y_, QPointF delta )
+{
+    const QPoint point( x_ * width(), y_ * height( ));
+    QWheelEvent* e = new QWheelEvent( point, delta.y(), Qt::NoButton,
```

Comment:
you're right, and this is removed in the second commit which introduces the touch points in deflect. I had put it there because it was an independent commit in the beginning, but maybe this was already a mistake. I can amend this commit.